REQUIREMENTS/connected-building-notes.md

```diff
@@ -14,4 +14,6 @@ Requirements:
 3. [Directory API and Information Model](connected-building-directory.md)
    - API requirements
    - Search response model
-   - Ontology, SAREF?
\ No newline at end of file
+   - SAREF vocabulary should be natively part of the TDs
+   - Directories should be part of the TDs, by being included in the TD ontology
```

Comment:
Is this referring to describing the directory with a TD?

Reply:
Yes, the second dash "Directories should be part of the TDs, by being included in the TD ontology" refers on the one hand that TDDs must be part of the TD, but also, that there should be a type to describe them.